docs/pull_request.md

```diff
@@ -0,0 +1,56 @@
+# Pull Request
+
+To contribute for CloudCV in EvalAI is a facile task. Please follow these steps in order to get started :-)
+
+**Step 1 : Fork**
+
+1. Fork the EvalAI repository from [EvalAI](https://github.com/Cloud-CV/EvalAI) link.
+
+**Step 2 : Picking up a issue**
+
+1. Pick up a suitable issue that you find will be facile for you to fix. Moreover, you can also 
+take the issues based on their labels. Almost every issue is marked with a label according to its difficulty.
```

Comment:
Replace `Almost every issue is marked with a label according to its difficulty.` with `All the issues are marked with a label according to its difficulty.`